btr-web/btr-common-components/components/bcros/inputs/Address/Line1Autocomplete.vue

```diff
@@ -98,6 +99,8 @@ const canadaPostApiKey = runtimeConfig.public.addressCompleteKey
 const suggestedAddresses: Ref<Array<CanadaPostApiFindResponseItemI>> = ref([])
 // @ts-ignore
 const line1: Ref<string> = ref(props.modelValue)
+const val = ref()
+val.value = { id: 'empty' }
```

Comment:
left this in by accident I think?

Reply:
Oh nvm combobox uses this or were you just experimenting with something? Doesn't look like it changes at all

Reply:
This one is required to have the value of the combobox refreshed first time. If you pass the value as prop. 
In short, its a workaround for how the headless UI keeps value and display value in input box :/